Provision/base.sql

```diff
@@ -65,7 +65,8 @@ CREATE TABLE IF NOT EXISTS summoners
 CREATE TABLE IF NOT EXISTS matches
 (
     id INT NOT NULL AUTO_INCREMENT,
-    role ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
+    match_id INT NOT NULL,
+    lane ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
```

Comment:
the role key returns a bunch of "none" with very little other info, just in my matchlist 1 of 5 are 'none' with a random 'duo', 'solo', and one odd 'duo_support". I do not believe the role to be a reliable return of information.